test/archivist/encodings.js

```diff
@@ -0,0 +1,27 @@
+const assert = require('assert');
+
+describe('Encoding', function () {
+	const encoders = require('../../lib/archivist/encoders');
+
+	it('can guess an encoding', function () {
+		assert.equal(encoders.guessEncoding('hello world'), 'utf8');
```

Comment:
Each assert being a single test here, it would be preferable to have a describe('can guess') with an it('') for each of those tests.

Reply:
Then I think it would make more sense to add message arguments to the assertion, no?

Reply:
The message wouldn't show on success cases.

Reply:
e.g. by looking at a test run output, we cannot see what has been tested (which is instructive for any newcomer to the project)